src/openst_deployer.js

```diff
@@ -0,0 +1,77 @@
+const Web3 = require('web3');
+const Package = require('@openstfoundation/openst.js');
+
+const logger = require('./logger');
+
+class OpenSTDeployer {
+  constructor(chainConfig) {
+    this.chainConfig = chainConfig;
+    this.auxiliary = {
+      web3: new Web3(chainConfig.auxiliaryWeb3Provider),
```

Comment:
Web3 should be injected with the proper provider. It is probably beneficial to inject the `Connection` (See `connected.run()`). See the latest code of JLP for that.